src/sdk/namespace_fs.js

```diff
@@ -509,12 +512,18 @@ class NamespaceFS {
     async read_object_md(params, object_sdk) {
         const fs_context = this.prepare_fs_context(object_sdk);
         try {
-            const file_path = this._get_file_path(params);
+            const file_path = await this._get_version_path(params, fs_context);
             await this._check_path_in_bucket_boundaries(fs_context, file_path);
             await this._load_bucket(params, fs_context);
             const stat = await nb_native().fs.stat(fs_context, file_path);
             if (isDirectory(stat)) throw Object.assign(new Error('NoSuchKey'), { code: 'ENOENT' });
-            return this._get_object_info(params.bucket, params.key, stat);
+            if (this.versioning === 'ENABLED' || this.versioning === 'SUSPENDED') {
```

Comment:
please define the following consts and use them instead of comparing the versioning mode with a string:
const VER_ENABLED = 'ENABLED';
const VER_SUSPENDED = 'SUSPENDED';
const VER_DISABLED = 'DISABLED';

Reply:
sure